autobotAI_integrations/integrations/symphony_summit/__init__.py

```
@@ -0,0 +1,293 @@
+        IntegrationCategory.NOTIFICATIONS_AND_COMMUNICATIONS.value
+    )
+    description: Optional[str] = (
+        "Symphony SummitAI ITSM platform — manage incidents, service requests, "
+        "and service catalog via the Summit REST WCF API."
+    )
+
+    @field_validator("base_url", mode="before")
+    @classmethod
+    def validate_base_url(cls, base_url) -> Optional[str]:
+        if not base_url or base_url == "None":
+            raise ValueError("Symphony Summit instance URL is required")
```

Ruff (0.15.20)

[warning] Avoid specifying long messages outside the exception class

(TRY003)